requirements/requirements.txt

```diff
@@ -422,7 +423,6 @@ requests==2.31.0
     #   -r /awx_devel/requirements/requirements.in
     #   adal
     #   azure-core
-    #   django-ansible-base
```

Comment:
I don't want to bikeshed on this too much, but requests is still a requirement of DAB resource_registry, which is an AWX requirement.

https://github.com/ansible/django-ansible-base/blob/devel/requirements/requirements_resource_registry.in

So yes, it is unexpected that this got removed. But I can't say why.

Reply:
I'll see if I can find the reason for that.